main.go

```diff
@@ -56,6 +57,12 @@ Misc options:
 		os.Exit(1)
 	}
 
+	rmqTimeout, err := strconv.Atoi(args["--rabbitmq-timeout"].(string))
```

Comment:
It's better to use `time.ParseDuration(args["--rabbitmq-timeout"])` [https://golang.org/pkg/time/#ParseDuration](https://golang.org/pkg/time/#ParseDuration) , because this method returns what you need - time duration, and it's not necessary convert to `time.Duration` in your further usages.

Reply:
Thank you for your feedback. I changed to time.ParseDuration(args["--rabbitmq-timeout"])